concepts/recursion/about.md

```diff
@@ -0,0 +1,191 @@
+# About
+
+Recursion is a way to repeatedly execute code inside a function through the function calling itself.
+Functions that call themselves are know as _recursive_ functions.
+Recursion can be viewed as another way to loop/iterate. 
+And like looping, a Boolean expression or `True/False` test is used to know when to stop the recursive execution.
+_Unlike_ looping, recursion without termination in Python cannot not run infinitely.
+Values used in each function call are placed in their own frame on the Python interpreter stack.
+If the total amount of function calls takes up more space than the stack has room for, it will result in an error.
+
+## Looping vs Recursive Implementation
+
+Looping and recursion may _feel_ similar in that they are both iterative.
+However, they both _look_ differently, both at the code level and at the implementation level.
+Looping can take place within the same frame on the call stack.
+It usually manages this by updating one or more variables to maintain state as it iterates.
+This is an efficient implementation, but it can be somewhat cluttered at the code level.
+Recursion, instead of updating state variables, can pass updated values as arguments to the next call of the same function.
```

Comment:
```suggestion

Recursion, rather than updating _variable state_, can pass _updated values_ directly as arguments to the next call (iteration) of the same function.
```